lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs

```diff
@@ -263,40 +264,110 @@ testRule({
 			fixed: 'a { top: calc(2px + 1px); }',
 			description: 'no space before or after operator',
 			warnings: [
-				{ message: messages.expectedAfter('+'), line: 1, column: 18, endLine: 1, endColumn: 19 },
-				{ message: messages.expectedBefore('+'), line: 1, column: 18, endLine: 1, endColumn: 19 },
+				{
+					message: messages.expectedAfter('+'),
+					line: 1,
+					column: 18,
+					endLine: 1,
+					endColumn: 19,
+					fix: { range: [17, 18], text: '+ ' },
```

Comment:
@romainmenke This doesn't look right, shouldn't the text be `' + '`?

Reply:
This rule produces two warnings for the same region.
One warning and one fix for the missing space before `+` and another for the missing space after `+`.

Because we only apply one fix at a time and prevent subsequent fixes on the same region you only get a fix for one of the reports.

So `test: '+ '` is correct for the current implementation for the rule.

Reply:
But in that case shouldn't the next case be `fix: { range: [17, 18], text: ' +' },` instead of `undefined`?

Reply:
Then there would be two overlapping `EditInfo` objects. Applying both of them would give incorrect outcomes. To avoid issues we skip calculating `fix` for the second warning.

When we do compute the second `EditInfo` the `text` would actually be ` + ` and not ` +` as the AST would still contain the mutation from the first fix. This is the "bleed" issue of doing multiple mutations on the same AST nodes.

Reply:
(thank you for carefully checking the tests on these changes)

Reply:
To also clarify here.

The case described above is not an issue that needs to be fixed.
It is a tradeoff in the current design for computing `EditInfo`.

This same tradeoff also exists in eslint, so this is acceptable.

In this specific rule it feels like a larger issue because this rule is so granular :)